ledger/sandbox-classic/BUILD.bazel

```diff
@@ -322,23 +322,16 @@ SERVERS = {
     },
 }
 
-# TODO append-only: only for temporary testing
-ONLY_POSTGRES_SERVER = {
-    "postgresql": {
-        "binary": ":sandbox-classic-ephemeral-postgresql",
-        "server_args": [
-            "--port=6865",
-            "--eager-package-loading",
-        ],
-    },
+# The in-memory ledger is deprecated, and does not support all operations (e.g., pruning)
+# TODO: remove this after removing the sandbox classic in-memory ledger
+NON_DEPRECATED_SERVERS = {
```

Comment:
I think the naming of non-deprecated is quite unusual. I would expect to find DEPRECATED_SERVERS, but the for the other ones I would use a different name. 
I'm a bit out of context but it seems to be specifically connected to the support for append-only schemas, so maybe a name more similiar to `APPEND_ONLY_SUPPORTED_SERVERS`, or `FULLY_SUPPORTED_SERVERS`

Reply:
:+1:

Context: sandbox has an in-memory ledger that we want to remove as soon as possible. We're not implementing new features (like participant pruning) on it, that's why conformance tests running on the in-memory ledger have to exclude some tests.